### PR TITLE
update root documentation

### PR DESCRIPTION
**Pull request by Ashar2shahid**
The amount of times I fail to do is embarrassing.
 Also @Siegrift encountered this in #330 so will close that as well

### CODE REVIEW

**Review by Siegrift — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ To install dependencies, run this at the repository root:
 yarn run bootstrap
 ```
 
+To build the project, run this at the repository root:
```
Comment (andreogle) [inline README.md:35]: I would include something about this being a required step for many (all?) commands

**Review by andreogle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bbenligiray) [reply README.md:35]: Good point, the reader may decide to be a smartass about it

**Comment by sitch** [comment]
Can you just add this as a `postinstall`?

So add to `package.json`:
```json
{
  "scripts": {
    "postinstall": "lerna bootstrap",
    ...
  }
}
```
Now after you call `yarn install` it will trigger the `yarn postinstall` automatically.  Not sure if this does exactly whats needed but worth a shot.  You can read more about pre/post scripts https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts

**Comment by andreogle** [comment]
I think ^ is probably a good idea. The `postinstall` should just be:

```json
{
  "scripts": {
    "postinstall": "yarn run build",
    ...
  }
}
```

**Comment by Ashar2shahid** [comment]
Yeah the post install script looks like the best way.

**Review by Siegrift — COMMENTED** [review]
@Ashar2shahid can you finish this? :)

**Comment by Ashar2shahid** [comment]
> @Ashar2shahid can you finish this? :)

just pushed a commit, tested it and it works well. You can have a look and then I'll merge it

**Review by andreogle — APPROVED** [review]

**Pull request merged**